src/Storages/VirtualColumnUtils.cpp

```diff
@@ -138,6 +142,35 @@ NameSet getVirtualNamesForFileLikeStorage()
     return getCommonVirtualsForFileLikeStorage().getNameSet();
 }
 
+std::string_view findHivePartitioningInPath(const String & path)
```

Comment:
Hive partition columns are not virtual anymore. If this function is really needed, let's move it to `HivePartitioningUtils`